docs/content/getting-started/database-component.md

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
Add the full files for both in examples repo and link to them so its easy to see the completed file.

Reply:
Added full files and linked relevant examples